doc/build.md

```diff
@@ -9,7 +9,7 @@ APIcast is released as [Docker image](https://docs.docker.com/engine/tutorials/d
 ## Dependencies
 
 APIcast uses LuaRocks to install dependencies. LuaRocks have to be properly configured,
-so it installs dependencies into correct path where OpenResty can see them. In the Docker image, rocks are installed into the application folder.
+so it installs dependencies into the correct path where OpenResty can see them. In the Docker image, rocks are installed into the application folder.
 Then this folder is added to the load path by `luarocks path`. 
```

Comment:
What does `this folder` refer to:
* the application folder of Docker Images
or
* the correct path where OpenResty sees the dependencies

Reply:
It is "the application folder of Docker Images " from what I understand and interpret.

Reply:
In this case, we need to restructure the content as the following:

```
APIcast uses LuaRocks, the package manager for Lua modules, to install dependencies. With a correct configuration, LuaRocks installs dependencies into the correct path where OpenResty can see them. 

For Docker images, LuaRocks is installed into the application folder. Then, `luarocks path` adds the application folder to the load path.
```

Does this make sense?

Reply:
Looks ok to me. Made the changes.